### ISSUE TITLE
More information to support chosing well

### ISSUE THREAD

**Issue by Gaibhne** [akullpp/awesome-java#859]
I love this repository, and have taken many suggestions from it. But one thing always bothers me when looking at it: when looking for a solution to a problem and not just window shopping, I am presented with a dozen or so items without any context as to the projects status. I am just as likely to click on a Spring sized project at random as I am on a project that has 15 commits, the last of which was "Sorry I haven't updated in a while, I promise I'm not gone" from three years ago ;-)

I am suggesting a rough indicator of a projects state and health in the overview; something like that could probably easily be generated at least for projects hosted on GitHub, GitLab and similar sites.

Useful metrics would be (to me) commits, releases, and time since last commit and/or release. Other metrics that could be useful, but might also overload the display, would be things like committers (we value projects with many active commiters over ones with single maintainers due to the [Bus Factor](https://en.wikipedia.org/wiki/Bus_factor)), commit # over the last six months (to indicate projects that are stalling) and dependency information.

A possible display might look something like this (taken from the current "REST Frameworks" section):

* ![#c5f015](https://via.placeholder.com/15/c5f015/000000?text=+) [[5,439/165/19 hours ago] - Dropwizard](https://github.com/dropwizard/dropwizard "In the last month, there were 64 commits by 3 authors") - Opinionated framework for setting up modern web applications with Jetty, Jackson, Jersey and Metrics.
* ![#c5f015](https://via.placeholder.com/15/c5f015/000000?text=+) [[1,972/177/6 days ago] - Elide](https://elide.io/ "In the last month, there were 52 commits by 18 authors") - Opinionated framework for JSON- or GraphQL-APIs based on a JPA data model.
* ![#c5f015](https://via.placeholder.com/15/c5f015/000000?text=+) [[1,214/63/9 days ago] - javalin](https://javalin.io/ "In the last month, there were 20 commits by 8 authors") - Javalin is just a few thousand lines of code on top of Jetty, which means its performance is almost equivalent to pure Jetty.
* [...]
* ![#f03c15](https://via.placeholder.com/15/f03c15/000000?text=+) [[5,200/73/**<font color='red'>Oct 13, 2018</font>**] - Rapidoid](https://www.rapidoid.org/ "In the last month, there were no commits!") - Simple, secure and extremely fast framework consisting of an embedded HTTP server, GUI components and dependency injection.

All the links have mouse-over hover text elements with additional information. If there was interest in this, I might be able and willing to provide a script or something to generate the information from web crawls, to be updated on regular intervals.

**Comment by akullpp** [comment]
Agreed, there should be some rules. If I have the time I'll create a script to do this.
